### PR TITLE
Improve documentation of the TextArea

### PR DESCRIPTION
**Pull request by dlouhak**
- Apply 🔤 order to the props and add `UNSAFE_*` props.
- Fix typo in TextArea's documentation – native attribute `maxlength` is valid instead of `maxLength`.

### CODE REVIEW

File: packages/web-twig/src/Resources/components/TextArea/README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Without lexer:
     id: "example",
     isRequired: true,
     label: "Label",
-    maxLength: 180,
+    maxlength: 180,
```
Comment (dlouhak) [inline packages/web-twig/src/Resources/components/TextArea/README.md:37]: This syntax is currently valid in Twig, but what about consistency with React where is used `maxLength`? We can unify naming as well, or can't we?

Comment (literat) [reply packages/web-twig/src/Resources/components/TextArea/README.md:37]: This is only a problem of the type, I think.

Comment (literat) [reply packages/web-twig/src/Resources/components/TextArea/README.md:37]: Please see https://github.com/lmc-eu/spirit-design-system/pull/896 where I solved this also for the web-react package.

**Review by literat — APPROVED** [review]

File: packages/web-twig/src/Resources/components/TextArea/README.md (hunk 1 above)
Comment (dlouhak) [reply packages/web-twig/src/Resources/components/TextArea/README.md:37]: Lovely, thank you. ❤️

File: packages/web-twig/src/Resources/components/TextArea/README.md [hunk 2]
```diff
@@ -46,24 +46,27 @@ Without lexer:
...
+| `isLabelHidden`     | `bool`                                                               | `false` | no       | If true, label is hidden                                                                                                           |
+| `isRequired`        | `bool`                                                               | `false` | no       | If true, TextArea is required                                                                                                      |
+| `label`             | `string`                                                             | —       | yes      | Label text                                                                                                                         |
+| `UNSAFE_label`      | `string`                                                             | —       | yes\*    | Unescaped label text                                                                                                               |
```
Comment (adamkudrna) [inline packages/web-twig/src/Resources/components/TextArea/README.md:61]: The footnote `*` is missing… 👀

**Review by adamkudrna — APPROVED** [review]
Cool, thank you! 🙏🏻

**Pull request merged**